docs/section19/19-14.md

```diff
@@ -0,0 +1,71 @@
+            - Reset is modeled with a relief fan in the proposed boolean variable: `is_modeled_with_relief_fan_in_proposed = false`   
+            - Get list of zones served by the hvac system: `zones_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]`  
+            - For each zone in the B_RMI check to see if the zone is served by an hvac system with return or relief fans in the proposed: `for zone in zones_list_hvac_sys_b:`  
+                - Get the analogous zone object from the proposed RMI (if this is not strictly the correct syntax please advise Weili): `zone_p = get_component_by_id(P_RMI,match_data_element(P_RMI,Zone,zone.id))`  
+                - Get list of the zone's supply, return, exhaust, and relief fan power modeled in the proposed: `modeled_fan_power_list_p = list(zone_supply_return_exhaust_relief_terminal_fan_power_dict[zone_p.id])`  
+                - Get the zone's total return fan kW, need to get the second element in the list of fan powers:`modeled_return_fan_power_p = modeled_fan_power_list_p[1]` 
+                - Get the zone's total relief fan kW, need to get the fourth element in the list of fan powers:`modeled_relief_fan_power_p = modeled_fan_power_list_p[3]` 
+                - Check if the total return fan power is greater than 0 in the proposed, if yes then set is_modeled_with_return_fan_in_proposed to true: `if modeled_return_fan_power_p > 0: is_modeled_with_return_fan_in_proposed = true`  
+                - Check if the total relief fan power is greater than 0 in the proposed, if yes then set is_modeled_with_relief_fan_in_proposed to true: `if modeled_relief_fan_power_p > 0: is_modeled_with_relief_fan_in_proposed = true`  
+                            
+
+            - Create fan system object: `fan_sys_b = hvac.fan_system`  
+            - Get list from dictionary of the hvac system's fan system's supply, return, exhaust and relief fan CFM: `fan_sys_CFM = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"]["Fan_CFM"])`  
+            - Get the hvac system total supply fan cfm, need to get the first element in the list of fan cfms (this assumes that all fans associated with the supply_fans object are in parallel (i.e., if multiple fans the cfm is additive.):`hvac_sys_total_supply_fan_cfm_b = fan_sys_CFM[0]` 
+            - Calculate 90% of the supply air quantity: `90_percent_supply_cfm = 90% * hvac_sys_total_supply_fan_cfm_b`   
+            - Calculate the supply CFM minus the minimum OA CFM for the fan system: `supply_minus_OA_cfm = hvac_sys_total_supply_fan_cfm_b - fan_sys_b.minimum_outdoor_airflow`  
+            - Check if is_modeled_with_return_fan_in_proposed == true, if so then set the modeled_cfm equal to the return cfm (second element in list of baseline hvac fan cfms): `if is_modeled_with_return_fan_in_proposed == true: modeled_cfm = fan_sys_CFM[1]`  
```

Comment:
Perhaps we need modeled_releif_CFM and modeled_return_CFM in case both are present.

Reply:
Do projects ever have both?